src/test/TestUtils/TestProjectFixture.cs

```diff
@@ -53,11 +53,13 @@ public class TestProjectFixture
             string dotnetInstallPath = null,
             string currentRid = null,
             string builtDotnetOutputPath = null,
-            string framework = "netcoreapp2.1")
+            string framework = null)
         {
             ValidateRequiredDirectories(repoDirectoriesProvider);
 
             _testProjectName = testProjectName;
+            _framework = framework ?? Environment.GetEnvironmentVariable("MNA_TFM");
+            Console.WriteLine($"TPF: MNA_TFM={_framework}");
```

Comment:
Do you intend this to be written all the time?  Or should it be removed before check in?

Reply:
Yeah that was for debugging I will remove.